Source/NSArrayController.m

```diff
@@ -585,6 +585,19 @@ - (void) encodeWithCoder: (NSCoder *)coder
     }
   else
     {
+      BOOL f;
+      f = _acflags.avoids_empty_selection;
+      [coder encodeValueOfObjCType: @encode(BOOL)
+                                at: &f];
+      f = _acflags.selects_inserted_objects;
```

Comment:
Why is `preservesSelection`missing here?

Reply:
Added missing data to non-keyed archiving logic.